src/OpenTelemetry/CHANGELOG.md

```diff
@@ -16,8 +16,13 @@
   ([#4375](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4375))
 
 * Tweaked the behavior of the `OpenTelemetryLoggerOptions.ParseStateValues`
-  flag, obsoleted `LogRecord.State` and `LogRecord.StateValues` properties, and
-  added `LogRecord.Body` and `LogRecord.Attributes` properties.
+  flag (attributes are now automatically included for most log messages by
+  default), obsoleted `LogRecord.State` and `LogRecord.StateValues` properties,
+  and added `LogRecord.Body` and `LogRecord.Attributes` properties.
+  ([#4334](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4334))
```

Comment:
nit: We need not add the PR link twice.

Reply:
I kind of did this on purpose. I think it reads more cleanly with conceptual changes broken out into their own simple(ish) bullets/lines. I could roll everything into one bullet, but having duplicate PR links doesn't seem like a big deal to me. LMK if you still want a change.

Reply:
I don't have a strong opinion here. I was suggesting we use multiple bullet points but add the PR link only once at the end of the last bullet point. Someone might complain about unclaimed bullet points then if they don't read the CHANGELOG the way we expect them to.